htdocs/content/themes/themosis-theme/style.css

```diff
@@ -1,11 +1,11 @@
 /*
-Theme Name: Themosis
-Theme URI: http://framework.themosis.com/
-Author: Julien Lambé
-Author URI: http://www.themosis.com/
-Description: The Themosis framework base theme.
-Version: 1.3.2
+Theme Name: Gumbo Millennium
+Theme URI: https://gumbo-millennium.nl/
+Author: Digitale Commissie
+Author URI: https://www.github.com/gumbo-millennium
+Description: The theme for the Gumbo Millennium.
```

Comment:
't is maar simpele description, maar moet dit niet: "The theme for the Gumbo Millennium **website**" zijn ?

Reply:
Yup